docs/install/upgrade-installation.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Upgrading your installation"
+weight: 21
+type: "docs"
+---
+
+To upgrade your Knative components and plugins, run the `kubectl apply` command
+to install the subsequent release. We recommend upgrading by a single minor
+version number. For example, if you have v0.6.0 installed, you should upgrade to
+v0.7.0 before attempting to upgrade to v0.8.0. To verify the version number you
+currently have installed, see
+[Checking your installation version](./check-install-version.md).
+
+If you installed Knative using the Operator plug-in, the upgrade process will
+differ. See []() to learn how to upgrade an install managed by the Operator.
+
+## Before you begin
+
+Before upgrading, there are a few steps you must take to ensure a successful
+upgrade process.
+
+### Identify breaking changes
+
+You should be aware of any breaking changes between your current and desired
+versions of Knative. Breaking changes between Knative versions are documented in
+the Knative release notes. Before upgrading, review the release notes for the
+target version to learn about any changes you might need to make to your Knative
+applications:
+
+- [Serving](https://github.com/knative/serving/releases)
+- [Eventing](https://github.com/knative/eventing/releases)
+- [Eventing-Contrib](https://github.com/knative/eventing-contrib/releases)
+
+Release notes are published with each version on the "Releases" page of their
+respective repositories in GitHub.
+
+### View current pod status
+
+Before upgrading, view the status of the pods for the namespaces you plan on
+upgrading. This allows you to compare the before and after state of your
+namespace. For example, if you are upgrading Knative Serving, Eventing, and the
+monitoring plug-in, enter the following commands to see the current state of
+each namespace:
+
+```bash
+kubectl get pods --namespace knative-serving
+```
+
+```bash
+kubectl get pods --namespace knative-eventing
+```
+
+```bash
+kubectl get pods --namespace knative-monitoring
+```
+
+### Upgrading plug-ins
+
+If you have a plug-in installed, make sure to upgrade it at the same time as
+you upgrade your Knative components. For example, if you have the
+monitoring plug-in installed, upgrade it alongside Knative Serving and Eventing.
+
+## Performing the upgrade
+
+To upgrade, apply the `.yaml` files for the new Knative version of your desired
+Knative components and features to your Kubernetes cluster, remembering to only
```

Comment:
Nit:
I think this is too long and complicated. It says "new version" (which can be confused with the latest version) and similarly, it refers to "desired" components and features (which you also used earlier to describe the target version of the upgrade "desired versions of Knative"). I think if we cut those out, then we dont need the gerund either and can shorten it?

"To upgrade, apply the `.yaml` files for the subsequent minor versions of all your installed Knative components and features.

Reply:
fixed